core/src/main/java/io/lionweb/lioncore/java/language/Containment.java

```diff
@@ -100,7 +100,7 @@ public void setSpecialized(@Nullable Containment specialized) {
 
   @Override
   public String toString() {
-    return "Containment{" + "name=" + getName() + ", " + "type=" + getType() + '}';
+    return super.toString() + "{" + "name=" + getName() + ", " + "type=" + getType() + '}';
```

Comment:
This is the same implementation as for `Reference`, so we can move it to `Link`

Reply:
Why don't construct a _qualifiedName_ like for `LanguageEntity` (for symmetry)?

Reply:
Moving to Link, and using qualifiedName with the same logic used in LanguageEntity. I will also use qualifiedName in Property.toString